NEW action button to print product barcodes

W1W1-M opened the pull request:
# NEW action button to print product barcodes

If barcode module is activated, user can open the barcode print sheet directly from product card. Print sheet is preselected with product.

<img width="1580" height="800" alt="Capture d’écran 2025-12-15 à 12 09 02" src="https://github.com/user-attachments/assets/5bed1873-8a94-4c3f-928c-f78bcb06ba5b" />

W1W1-M commented:
> My undestanding is that button is not a workflow action on status but a link to anohter page that is preselected. So i think it would be better to replace the button, with a small link "Print barcode" just after the "Barcode value" like we have on this screenshot for the "Stock at date" or "Virtual stock" just after the value of stock : <img alt="image" width="1595" height="427" src="https://private-user-images.githubusercontent.com/883887/526607620-a421717d-9805-4706-b1ca-28ad88b4621f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTY2MTAsIm5iZiI6MTc2NTgxNjMxMCwicGF0aCI6Ii84ODM4ODcvNTI2NjA3NjIwLWE0MjE3MTdkLTk4MDUtNDcwNi1iMWNhLTI4YWQ4OGI0NjIxZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNjMxNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NTNiMGRmZWFiNjE0YWQ4NDllMjMzNDYzMjAwNmM0NWEwMjhmNjA3NDI2Y2ZmZDAxMzNkZDRhODY4NGEwYWRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.CQV2NCSA6hbOVGnlxGUZpkXD2uppKrZntUzEHVhnFSA">

@eldy Thanks for feedback. Refactored action button to link on barcode value row:
<img width="1590" height="800" alt="Capture d’écran 2025-12-15 à 17 32 28" src="https://github.com/user-attachments/assets/7f1b1d12-145b-414a-bf2a-b951d0d24ba7" />

eldy commented:
Thanks. Pr is postponed for the moment, will be validated once branch v23 has been created (next step will be to add a larger space and add the picto used for links for the look and feel v24)...

braito4 committed:
Fix SMTP OAuth refresh scopes (#36950)

Recreate the OAuth service with its configured scopes when refreshing an access token (required by some providers like Microsoft v2).

Also keep the previous refresh token when the refreshed token does not include one (e.g. Google returns it only once).

Co-authored-by: caminotravelcenter <caminotravelcenter@localhost>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

aspangaro committed:
Move module asset to stable (#36983)

eldy committed:
Code comment

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Clean code

eldy committed:
Debug v23 - Fix trans

eldy committed:
Trans

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v23

eldy committed:
Debug v23

noec764 committed:
FIX: Add missing event counter on Agenda tab (#36972)

Co-authored-by: Noé <noe@scopen.fr>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Trans

frederic34 committed:
display color selected in dict (#36500)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

aspangaro committed:
Update VAT rate by default (#36988)

* Update VAT rate for Nederland

* Update VAT rate for Italy

eldy committed:
ci

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v23

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
CSS v23

eldy committed:
Debug v23

eldy committed:
CSS v23

eldy committed:
Code comment

eldy committed:
Fix ci

atm-vincent-p committed:
Automatically send the invoice generated from a template DB (#36947)

* Update DB

* Add flag auto send

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Fix CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Code comment

eldy committed:
ci

eldy committed:
ci

eldy committed:
CI

eldy committed:
ci

braito4 committed:
FIX|Fix supplier payment presend mail/PDF (#36939)

- Keep the presend form open after applying an email template.

- Use getDolGlobalString('SUPPLIER_PAYMENT_ADDON_PDF') (no direct ->global access).

- Ensure a PDF model is selected (fallback to first available or standard_supplierpayment) so receipt generation/attachment works.

Signed-off-by: Camino Travel Center <noreply@caminotravelcenter.com>
Co-authored-by: Camino Travel Center <noreply@caminotravelcenter.com>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Debug v23

eldy committed:
Merge branch '22.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Set default value to on

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Trans

eldy committed:
Prepare v23

eldy committed:
Add the registration page

eldy committed:
WIP LNE

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Default value

eldy committed:
Fix trans and warning

eldy committed:
Doc

eldy committed:
Debug v23

eldy committed:
Debug v23

eldy committed:
Debug v23

eldy committed:
Doc

Hystepik committed:
New import Leave Balance (Datastructure) (#37001)

* New import Leave Balance (Datastructure)

* fix

---------

Co-authored-by: Lucas Marcouiller <lmarcouiller@dolicloud.com>

mdeweerd committed:
Qual: Update Phan baseline.txt with reduced issue counts (#36996)

This commit updates the Phan baseline.txt file.

eldy committed:
Better log

eldy committed:
Log

eldy committed:
Better error management

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Doc

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Fix error message

eldy committed:
CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Try to fix CI

frederic34 committed:
fix phan (#37007)

eldy committed:
CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

frederic34 committed:
fix phan (#37008)

PhanTypeMismatchArgumentNullable Argument 3 ($txt) is $productlot->batch of type ?string but \TCPDF::MultiCell() takes string defined at htdocs/includes/tecnickcom/tcpdf/tcpdf.php:5871 (expected type to be non-nullable)

eldy committed:
CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

aspangaro committed:
NEW Add new EUID number (#36997)

eldy committed:
Doc

eldy committed:
ci

eldy committed:
Fix CI

braito4 committed:
Improve OAuth HTTP error details (#36951)

Keep throwing TokenResponseException on HTTP 4xx/5xx while also capturing the response body (ignore_errors) to include a short snippet for admins.

Co-authored-by: caminotravelcenter <caminotravelcenter@localhost>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

Hystepik committed:
NEW add import of leave balance (Implementation) (#37011)

* New add import Leave Balance

* add eof

* Update modHoliday.class.php

---------

Co-authored-by: Lucas Marcouiller <lmarcouiller@dolicloud.com>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

frederic34 committed:
NEW add image format avif (#37014)

* add format avif

* Update functions.lib.php

eldy committed:
Exclude non interesting warning

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

eldy committed:
CI

eldy committed:
CI

eldy committed:
CI

eldy committed:
Trans

eldy committed:
CI

eldy committed:
CI

eldy committed:
CI

hansemschnokeloch committed:
Fix typo in phpstan.neon.dist for isset() rule (#37018)

See https://github.com/Dolibarr/dolibarr/actions/runs/21444577128/job/61756970301?pr=37017

hansemschnokeloch committed:
Remove jstz.min.js from JavaScript array (#37017)

jstz.min.js was deleted in https://github.com/Dolibarr/dolibarr/commit/5599ac733b07d990d64e3ff18b30b3dca16d5b86

hansemschnokeloch committed:
Remove jstz.min.js from JavaScript array (#37017)

jstz.min.js was deleted in https://github.com/Dolibarr/dolibarr/commit/5599ac733b07d990d64e3ff18b30b3dca16d5b86

atm-jonathan committed:
prevent ST_AsWKT() (#37013)

* prevent ST_AsWKT()

* Update commonobject.class.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

atm-jonathan committed:
prevent ST_AsWKT() (#37013)

* prevent ST_AsWKT()

* Update commonobject.class.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Try to fix phan

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
CI

eldy committed:
CI

eldy committed:
ci

eldy committed:
ci

eldy committed:
css

mdeweerd committed:
qual: Update french texts (comments) to english (#37009)

* qual: Update french texts (comments) to english

* Qual: Update comment to english in card-rec.php and card.php

Translation of comments in the files card-rec.php and card.php.

* Qual: Translate comments to English

* Qual: Translate comments to English

* Fix space into tab

* Fix spelling

* Qual: Translate comments to English

* Update website.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
ci

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Clean code

eldy committed:
Fix test

eldy committed:
Merge manually changes from #37012 that seems good.

eldy committed:
Debug v23

eldy committed:
Log

eldy committed:
More robust phpunit

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

hansemschnokeloch committed:
Populate openid data from wellknow url (#37023)

* Delete inexistant file

File jstz.min.js was deleted in
https://github.com/Dolibarr/dolibarr/commit/5599ac733b07d990d64e3ff18b30b3dca16d5b86

* Populate openid parameters from wellknow url

eldy committed:
CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v23 - fix code for rounding

sonikf committed:
MCP server sql part (#37025)

* Create llx_ai_request_log.sql

* Create llx_ai_request_log.key.sql

* Update 23.0.0-24.0.0.sql

* Update llx_ai_request_log.sql

* Update 23.0.0-24.0.0.sql

* Remove foreign key constraint from llx_ai_request_log

Removed foreign key constraint for fk_user in llx_ai_request_log.

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v24

eldy committed:
Fix syntax error

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

defrance committed:
NEW allow to disable freezone product on takepos (#37004)

* allow to disable freezone product on takepos

* Conditionally add FreeZone product to menus

* Add NoFreeZoneProduct language entry

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Fix label of dispute

Thatoo committed:
[database] Minimum amount and subscription's amount formula description (#37021)

* Remove jstz.min.js from JavaScript array (#37017)

jstz.min.js was deleted in https://github.com/Dolibarr/dolibarr/commit/5599ac733b07d990d64e3ff18b30b3dca16d5b86

* prevent ST_AsWKT() (#37013)

* prevent ST_AsWKT()

* Update commonobject.class.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

* css

* Update llx_adherent_type.sql

add minimumamount and amountformuladescription rows

* Update 22.0.0-23.0.0.sql

add minimumamount and amountformuladescription column

---------

Co-authored-by: hansemschnokeloch <hansemschnokeloch@users.noreply.github.com>
Co-authored-by: atm-jonathan <146709163+atm-jonathan@users.noreply.github.com>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

Boushabazakaria committed:
NEW: Display thirdparty name with ref in supplier orders linked objects (#36952)

Co-authored-by: Zakaria Boushaba <z.boushaba@vold.africa>

eldy committed:
css

eldy committed:
Debug v23

eldy committed:
Fix type

eldy committed:
CI

eldy committed:
CI

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

eldy committed:
Test without file list

eldy committed:
CI

defrance committed:
Add api document management for holiday (#36915)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

aspangaro committed:
NEW Invoice - List - use select2 multiselect for status (#36834)

* NEW Invoice - List - use select2 multiselect for status

* Optimize

* CI

* CI

* Review

* Duplicate line and last problem

* CI

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
FIX Pb in total price of line when adding the discount of a down payment
when unit price was using a high accuracy.

eldy committed:
Clean code

eldy committed:
try to fix phan

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix phan

eldy committed:
Clean code

eldy committed:
Clean code

eldy committed:
Debug v23

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix LDAP

frederic34 committed:
fix facture.class.php (#37031)

eldy committed:
Debug v23

mdeweerd committed:
Qual: Update french comments with 'nombre' (#37032)

Update french comments with 'nombre' in their phrasing

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

frederic34 committed:
fix phan (#37029)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

frederic34 committed:
fix phan (#37030)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Doc

eldy committed:
Doc comment

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
FIX When bank direct debit SEPA ref is > 99

eldy committed:
Fix when there is more than 99 direct debit in same month

eldy committed:
Fix when there is more than 99 direct debit in same month

eldy committed:
Debug v24

eldy committed:
Debug v24

eldy committed:
CSS

eldy committed:
FIX Division by zero

eldy committed:
FIX Division by zero

eldy committed:
More log

eldy committed:
More log

eldy committed:
More log

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

eldy committed:
More Log

eldy committed:
Fix IPN payment

eldy committed:
Fix IPN

eldy committed:
Log

eldy committed:
Clean code

eldy committed:
Debug v23 - Fix ipn for dispute and withdraw funds.

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

frederic34 committed:
fix CI (#37033)

frederic34 committed:
fix CI (#37033)

eldy committed:
CI

eldy committed:
Merge branch '22.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '22.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

frederic34 committed:
fix french doc (#36897)

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix

* fix

* fix

* fix

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* Update html.form.class.php

* Update html.form.class.php

* Update index.php

* fix

* fix

* fix

* fix

* fix

* fix

* fix

* fix

* Update modules_facture.php

* Update phpstan.neon.dist

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

aspangaro committed:
Qual: Translate French comments to English (#37044)

* Qual: Translate French comments to English

* Qual: Translate French comments to English

mdeweerd committed:
Qual: Translate French comments to English ("niveau") (#37041)

Translate French comments to English (comments with "niveau")

frederic34 committed:
fix french doc (#37040)

defrance committed:
warning error if no societe->id defined (#37039)

frederic34 committed:
fix french doc (#37037)

* fix french doc

* Update dispatch.php

* Update dispatch.php

* Update graph.php

frederic34 committed:
fix french doc (#37047)

* fix french doc

* Update index.php

* Update index.php

eldy committed:
Fix CI

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

eldy committed:
CI

eldy committed:
Fix CI

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

Thatoo committed:
Minimum amount and subscription's amount formula description (#37006)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

aspangaro committed:
Warning on export/import - Add isset() protection in Export::load_arrays() & Import::load_arrays() (#37043)

* Warning on export - Add isset() protection in Export::load_arrays()

* Warning on import - Add isset() protection in Import::load_arrays()

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Clean comment

aspangaro committed:
Warning on account model (#37042)

* Error on account model

* Simplify rowid assignment in accountmodel.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

vmaury committed:
Fix #35394 Display public note on first page (#36993)

* Display public note (an incoterms and sales rep signat) only on real first page

* Display public note (an incoterms and sales rep signat) only on real first page

---------

Co-authored-by: vmaury <vmaury@vmaury-Lafite-Pro-16-AMD>
Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

jyhere committed:
FIX: Add htmlname and selectedrate to parameters array for hook (#36998)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

jyhere committed:
FIX: Add htmlname and selectedrate to parameters array for hook (#36998)

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Fix CI

eldy committed:
Fix CI

eldy committed:
Fix CI

eldy committed:
Fix ci

aspangaro committed:
Qual: Translate French comments to English (#37052)

mdeweerd committed:
Qual: Translate French 'valeur' in comments to English (#37051)

aspangaro committed:
NEW Accountancy - Select between accrual accounting and cash accounting (#37050)

mdeweerd committed:
Qual: Translate french 'avec' comments to English (#37049)

# Qual: Translate french 'avec' comments to English

leninrivas committed:
FIX divisa rate indirect to direct because Dolibarr use indirect for default (#37046)

* FIX rate_indirect to rate_direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect	: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* FIX divisa rate indirect to direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* FIX divisa rate indirect to direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* FIX divisa rate indirect to direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* FIX divisa rate indirect to direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* Divisa rate direct

Direct	: 	1 Divisa Currency = X Currency Main.
Indirect: 	1 Currency Main = X Divisa Currency.
Then for Dolibarr use is Indirect for default

* Add col rate_direct because Dolibarr use for.default rate indirect

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
CI

frederic34 committed:
Add thumbnails for avif (#37045)

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* vignette for avif

* fix

* fix

* fix

* fix

* fix

* fix

* fix

* fix

* fix

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
FIX #36892

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

thomas-Ngr committed:
NEW : Manage discount for all lines for supplier elements. (#36954)

* NEW : Manage remise for all lines for supplier elements.

* Cast remise_percent to float in updateline call

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

mapiolca committed:
Fix Signature position (#37048)

Move up the position of the signature to fix its explosion on 3 pages

frederic34 committed:
remove french doc and fix travis migration missing (#37053)

* fix french doc

* fix french doc

* fix french doc

* fix french doc

* Clean up comments in card.php

Removed unnecessary comments from card.php

---------

Co-authored-by: Laurent Destailleur <eldy@destailleur.fr>

eldy committed:
Debug v23

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into 23.0

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix CSS

eldy committed:
Fix CSS

eldy committed:
Fix sql syntax error

eldy committed:
Fix sql syntax error

hansemschnokeloch committed:
Add optionnal img to openid login (#37060)

frederic34 committed:
fix french doc (#37056)

* fix french doc

* fix french doc

* fix french doc

* doc translation

* doc translation

* doc translation

* doc translation

* doc translation

defrance committed:
load tables in init for create categorie table associated (#37059)

eldy committed:
Fix situation invoice

eldy committed:
Fix situation invoice

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v23

eldy committed:
Link and filter on ountry for holiday types

eldy committed:
Link and filter on ountry for holiday types

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

eldy committed:
Trans

eldy committed:
Debug v23

eldy committed:
css

eldy committed:
CSS on current day

eldy committed:
Merge branch '23.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
More complete message

eldy committed:
Make array of dispute status a shared constant

eldy committed:
Debug v23

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button

W1W1-M committed:
Merge branch 'develop' into new-product-print-barcodes-button
